pkg/codegen/testing/test/testdata/azure-native-nested-types/nodejs/types/documentdb/output.ts

```diff
@@ -0,0 +1,36 @@
+// *** WARNING: this file was generated by test. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+import * as pulumi from "@pulumi/pulumi";
+import * as inputs from "../../types/input";
```

Comment:
Cool! So our next difficult task will be to make these imports here more precise 😬

Reply:
Yes! Indeed it is!